Makefile

```diff
@@ -499,9 +500,31 @@ $(DIST_DIR)/CHANGELOG.md: $(ROOT_DIR)/cliff.toml bin/git-cliff | $(DIST_DIR)
 
 .PHONY: multimod-verify
 multimod-verify: bin/multimod
-	@echo "Validating versions.yaml file"
+	$(info --- Validating versions.yaml file)
```

Comment:
Nope. We use the builtin [`info`](https://www.gnu.org/software/make/manual/make.html#index-info) function of Make to print messages to the console instead of calling shell command `echo`.